client/src/utils/isEmpty.js

```diff
@@ -0,0 +1,7 @@
+const isEmpty = value =>
```

Comment:
@prateekinstillion , bro this is doing too many things for a good arrow function. Please try to improvise towards more readability here.

Reply:
It is generic bro. It can be used as and when needed